### PR TITLE
Formating 3rd-party add,list,remove command output

### PR DESCRIPTION
**Pull request by castulo**
This commit makes a few small changes to the output of the commands to
make it look cleaner and to be consistent with the output of other swupd
commands.

Signed-off-by: Castulo Martinez <castulo.martinez@intel.com>

### CODE REVIEW

File: src/3rd_party_add.c [hunk 1]
```diff
@@ -180,16 +180,20 @@ enum swupd_code third_party_add_main(int argc, char **argv)
...
-		info("There was an error adding 3rd-party repository %s, reverting...\n", name);
 		set_path_prefix(path_prefix);
 		set_state_dir(state_dir);
 		ret = remove_repo(name);
-		if (!ret) {
-			info("Operation reverted\n");
-		} else {
-			error("There was an error removing the repository (errno: %d)\n", ret);
+		if (ret) {
+			error("There was an error removing the corrupt repository (errno: %d)\n\n", ret);
```
Comment (karthikprabhuvinod) [inline src/3rd_party_add.c]: Getting a little verbose .. how about "error removing the corrupt repository"

File: src/3rd_party_repos.c [hunk 2]
```diff
@@ -183,15 +183,15 @@ static int overwrite_repo_config_file(struct list *repos)
 	// open it in trunc mode as we re-write all the contents again
 	fp = fopen(repo_config_file_path, "w");
 	if (!fp) {
-		error("Impossible to write to file %s", repo_config_file_path);
+		error("Impossible to write to file %s\n", repo_config_file_path);
```
Comment (karthikprabhuvinod) [inline src/3rd_party_repos.c]: Unable to write to the file?

File: src/3rd_party_repos.c [hunk 3]
```diff
@@ -183,15 +183,15 @@ static int overwrite_repo_config_file(struct list *repos)
...
 		goto exit;
 	}
 
 	repo = repos;
 	while (repo) {
 		ret = write_repo(fp, repo->data);
 		if (ret < 0) {
-			error("Write error when trying to write to %s", repo_config_file_path);
+			error("Write error when trying to write to %s\n", repo_config_file_path);
```
Comment (karthikprabhuvinod) [inline src/3rd_party_repos.c]: Error when trying to write to

**Comment by karthikprabhuvinod** [comment]
@castulo : left you with some suggestions for wording the errors.

File: src/3rd_party_add.c (hunk 1 above)
Comment (castulo) [reply src/3rd_party_add.c]: Hmm, that would look strange cause the error() function is already going to add "Error: ", so to would read " Error: error removing the corrupt repository".
I could change it to something like "The corrupt repository failed to be removed", what do you think?

File: src/3rd_party_repos.c (hunk 2 above)
Comment (castulo) [reply src/3rd_party_repos.c]: Well, my commit didn't really modified that message I only added a missing line break, I'm not too concerned with this error message cause I don't expected it to be shown a lot. Having said that, sure, I could change it to what you suggests, the current error message seems a little odd.

File: src/3rd_party_repos.c (hunk 3 above)
Comment (castulo) [reply src/3rd_party_repos.c]: Same problem as before, starting the sentence with "error" will result in "Error": error when..."

File: src/3rd_party_add.c (hunk 1 above)
Comment (karthikprabhuvinod) [reply src/3rd_party_add.c]: I would put the event first... but upto you.

how abt you?
Failed to remove corrupt repository

File: src/3rd_party_repos.c (hunk 3 above)
Comment (karthikprabhuvinod) [reply src/3rd_party_repos.c]: "Failed to write to %s" in that case

**Pull request merged**